### DISCUSSION TITLE
GitHub Student Developer Pack

### DISCUSSION THREAD

**Discussion by Nisal-N-Narasinghe** [community/community#127612] [GitHub Education]
### Hi everyone,

I applied for the GitHub Student Developer Pack. Now it says "Congrats! Your request for [@Nisal-N-Narasinghe](https://github.com/Nisal-N-Narasinghe) was approved and your benefits will be available soon." But I still got nothing.

**Comment by davevad93** [comment]
Hi @Nisal-N-Narasinghe ,

You have to wait till the message changes to "`Congrats! Your request for @Nisal-N-Narasinghe was approved and your benefits are now available.`". Be patient because the waiting time may vary from a few days to a few weeks.
